### PR TITLE
Homepage link component

### PR DESCRIPTION
**Pull request by wpears**
Depends on #71 
Small Homepage link component, rendered as a part of the AppContainer

### CODE REVIEW

File: src/less/components/homeLink.less [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.HomeLink {
```
Comment (awolfe76) [inline src/less/components/homeLink.less]: I think we should just remove this `.less` file for now. This should probably render in place acting almost like a bread crumb. See this example - http://beta.consumerfinance.gov/the-bureau/leadership-calendar/

home-link

We can add styling later ... to add the icon if needed.

File: src/less/app.less [hunk 2]
```diff
@@ -14,6 +14,7 @@
 @import (less) "components/institutionStatus.less";
 @import (less) "components/resubmit.less";
 @import (less) "components/uploadForm.less";
+@import (less) "components/homeLink.less";
```
Comment (awolfe76) [inline src/less/app.less]: Not needed if we remove the `.less` file.

File: src/js/HomeLink.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+var React = require('react');
+var Link = require('react-router').Link;
+
+var HomeLink = React.createClass({
+  render: function(){
+    return <Link className="HomeLink" to="/">Home</Link>
```
Comment (awolfe76) [inline src/js/HomeLink.jsx:6]: We can remove the `className="HomeLink"` here. We _may_ add another class at some point to add an icon.

Comment (wpears) [inline src/js/HomeLink.jsx:6]: I kinda like having the components name as the classname on the node it returns. Seems convenient for debugging and such, and any specific styling outside the general grid system, etc. Doesn't really cost us anything.

Comment (awolfe76) [inline src/js/HomeLink.jsx:6]: :+1: on the classname!

Comment (wpears) [inline src/js/HomeLink.jsx:6]: Ticketed as #74

**Comment by awolfe76** [comment]
:100:

**Pull request merged**